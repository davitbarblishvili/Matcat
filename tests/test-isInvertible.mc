// Author: Davit
func main() int {
    matrix a;
    a = [[3,3],[5,5]];
    isInv(a);

    return 0;
}