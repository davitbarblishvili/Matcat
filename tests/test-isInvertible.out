This matrix is not invertible